crates/bcr-wallet-core/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "bcr-wallet-core"
+version = "0.1.0"
+edition = "2024"
```

Comment:
+license=MIT (also in workspace Cargo.toml)

or directly use 

```
version.workspace = true
edition.workspace = true
license.workspace = true
```

and put all int ehr workspace